FIXTURES.md

```diff
@@ -0,0 +1,14 @@
+# invalid-no-layers
+
+Empty protocol buffer with no layers array
+
+> A Vector Tile SHOULD contain at least one layer.
+> ([reference](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/vector_tile.proto#L75))
+
+# valid-single-point-no-id
+
+Has a single feature without an ID.
+
+> A feature MAY contain an id field. If a feature has an id field, the value
+> of the id SHOULD be unique among the features of the parent layer
+> ([reference](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md#42-features))
```

Comment:
I think we should move descriptions for each fixture into the fixture, per `We should think about some way of structuring the test vector` at https://github.com/mapbox/vector-tile/issues/20#issuecomment-315067795. We could do a folder per fixture, and then key files inside could:

 - describe it
 - have the code to generate it
 - have the fixture itself

See https://github.com/osmcode/osm-testdata/tree/master/grid/data/3/302 for inspiration.